core/src/main/kotlin/com/bitwarden/core/util/IntExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package com.bitwarden.core.util
+
+import android.app.PendingIntent
+import android.os.Build
+import com.bitwarden.annotation.OmitFromCoverage
+
+/**
+ * Starting from an initial pending intent flag. (ex: [PendingIntent.FLAG_CANCEL_CURRENT])
+ */
+@OmitFromCoverage
+fun Int.toPendingIntentMutabilityFlag(): Int =
```

Comment:
I'm not a fan of it, but I had to move this to `core` because it's used in `ui` and `data` packages. I'd prefer to either make it private in the few places it's used. Open to suggestions about this.